srtcore/README.md

```diff
@@ -6,6 +6,7 @@ and internally by the library, this directory also contains:
 
  - common files: usually header files, which can be used also by other projects,
 even if they don't link against SRT
+
```

Comment:
According to the [markdown-style-guide](https://cirosantilli.com/markdown-style-guide#empty-lines-inside-lists), if one or more list-items in a list spans more than one line, the list is required to have blank lines between each item. And otherwise, there should not be blank lines between items.